src/app/(sidebar)/my-recruit/api/useGetAllRecruits.ts

```diff
@@ -0,0 +1,27 @@
+import { http } from '@/apis/http';
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { RecruitCard } from '@/app/(sidebar)/my-recruit/type';
+import { ALL_RECRUITMENT } from '@/app/(sidebar)/my-recruit/containers/components/SeasonDropdownContent';
+
+type Request = { season: string };
+
+type Response = { data: RecruitCard[] };
+
+export const GET_RECRUITS = 'recruits';
+
+function getAllRecruits() {
+  return http.get({ url: '/recruits' });
```

Comment:
```suggestion
  return http.get<Response>({ url: '/recruits' });
```

이렇게 사용하면 tanstack-query의 반환값 타입을 명시하지 않아도 가능해요